test/PriceOracle.test.ts

```diff
@@ -49,6 +49,13 @@ describe('PriceOracle', function () {
       const postageStamp = await priceOracle.postageStamp();
       expect(postageStamp).to.be.eq((await ethers.getContract('PostageStamp')).address);
     });
+
+    it('current price should be equal to minimum price', async function () {
```

Comment:
Maybe add a test case that ensures that the `currentPrice` is never less than `minimumPrice`.

Reply:
Done.

Reply:
@significance @istae can you please maybe look into it and if its a bug we can create a new issue